### PR TITLE
test: accept exact queue-delay boundary (>=) in seq-batcher timing as…

### PR DESCRIPTION
**Pull request by akhilraj9**
#### What does the PR do?
Fix a strict-inequality timing boundary in the sequence batcher test helper that
caused `test_queue_delay_half_min_util` and `test_queue_delay_full_min_util` to
fail deterministically on the RTX50/RHEL runner. Changes `>` to `>=` at 4 sites
in `qa/common/sequence_util.py` so that a response arriving in exactly the
configured queue delay (3000 ms) is correctly accepted.

#### Checklist
- [x] PR title reflects the change and is of format `<commit_type>: <Title>`
- [x] Changes are described in the pull request.
- [x] Related issues are referenced.
- [ ] Populated [github labels](https://docs.github.com/en/issues/using-labels-and-milestones-to-track-work/managing-labels) field
- [x] Added [test plan](#test-plan) and verified test passes.
- [x] Verified that the PR passes existing CI.
- [ ] Verified copyright is correct on all changed files.
- [ ] Added _succinct_ git squash message before merging [ref](https://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html).
- [x] All template sections are filled out.
- [ ] Optional: Additional screenshots for behavior/output changes with before/after.

#### Commit Type:
- [x] test

#### Related PRs:
None

#### Where should the reviewer start?
`qa/common/sequence_util.py` — 4 one-line changes at lines 677, 710, 886, 1103.
Each changes `(...) > gt_ms` to `(...) >= gt_ms`. No logic change beyond the
boundary condition. All other files are untouched.

#### Test plan:
1. Run `L0_sequence_batcher--RTX50--rhel` and `L0_sequence_batcher_shm--RTX50--rhel`
   on the RTX50 RHEL runner with `TRITON_RHEL=1`.
2. Confirm `test_queue_delay_half_min_util` and `test_queue_delay_full_min_util`
   pass (`num_failures=0`, `num_errors=0`).
3. Confirm with repetition if this test passes on multiple runs — to check for flakiness.

- CI Pipeline ID: 59190261

#### Caveats:
Verification depends on TRI-1608 (gRPC/protobuf build fix) being merged to `main`
across `common`, `core`, and `third_party` repos. 

#### Background
The server correctly waits exactly `max_queue_delay_microseconds` (3000 ms) before
dispatching when `minimum_slot_utilization` cannot be met. Time is measured as
`int(round(time.time()*1000))` — whole milliseconds. On the low-jitter RTX50/RHEL
runner, elapsed time rounds to exactly 3000 ms, so `3000 > 3000 → False →
AssertionError`. On slower machines, OS jitter adds ≥ 0.5 ms and it rounds to
3001+ → passes. The strict `>` has been in the code since 2019 and was newly
exposed by the faster RTX50 hardware. `>=` is correct — it asserts "waited *at
least* the configured delay" while still catching too-early responses.

-------------------------------------------------------------------------------------------------------------

**Log evidence — BEFORE fix (4 consecutive nightly failures, shm job = TRI-1379):**

The exact assertion error, identical across all 4 failing runs:

raw_2.txt (2026-05-16) — L0_sequence_batcher_shm--RTX50--rhel
line 11669:  *** Test test_queue_delay_half_min_util Failed
line 11757:  *** Test test_queue_delay_full_min_util Failed

raw_3.txt (2026-05-16) — L0_sequence_batcher_shm--RTX50--rhel
line 11669:  *** Test test_queue_delay_half_min_util Failed
line 11757:  *** Test test_queue_delay_full_min_util Failed

raw_5.txt (2026-05-13) — L0_sequence_batcher_shm--RTX50--rhel
line 11717:  *** Test test_queue_delay_half_min_util Failed
line 11812:  *** Test test_queue_delay_full_min_util Failed

raw_6.txt (2026-07-16) — L0_sequence_batcher_shm--RTX50--rhel
line 12230:  *** Test test_queue_delay_half_min_util Failed
line 12308:  *** Test test_queue_delay_full_min_util Failed
The measured value was exactly 3000 ms against a threshold of 3000 ms in every
single run — deterministic boundary hit, not random flakiness.

-------------------------------------------------------------------------------------------------------------

**Log evidence — AFTER fix (pipeline #59190261, 2026-07-23):**

raw_8.txt — CI_JOB_NAME=L0_sequence_batcher--RTX50--rhel (TRI-1372)
line 12043:  python3 sequence_batcher_test.py SequenceBatcherTest.test_queue_delay_half_min_util
line 12059:  num_failures=0  num_errors=0  num_tests=1  ✅
line 12153:  python3 sequence_batcher_test.py SequenceBatcherTest.test_queue_delay_full_min_util
line 12167:  num_failures=0  num_errors=0  num_tests=1  ✅
line 12403:  *** Test Passed ***

raw_9.txt — CI_JOB_NAME=L0_sequence_batcher_shm--RTX50--rhel (TRI-1379)
line 12332:  python3 sequence_batcher_test.py SequenceBatcherTest.test_queue_delay_half_min_util
num_failures=0  num_errors=0  num_tests=1  ✅
line 12442:  python3 sequence_batcher_test.py SequenceBatcherTest.test_queue_delay_full_min_util
num_failures=0  num_errors=0  num_tests=1  ✅
line 12606:  *** Test Passed ***

-------------------------------------------------------------------------------------------------------------

#### Related Issues:
- Closes TRI-1372
- Closes TRI-1379

### CODE REVIEW

**Comment by pskiran1** [comment]
@akhilraj9, please update the copyright to `2019-2026`.

**Review by pskiran1 — DISMISSED** [review]

File: qa/common/sequence_util.py [hunk 1]
```diff
@@ -40,6 +40,11 @@
 import tritonclient.http as httpclient
 from tritonclient.utils import *
 
+# QA-suite convention: tritonclient.utils is imported via `*` here and in 20+
+# other QA files. Silence flake8's star-import warnings (F403/F405) for this
+# file only; proper cleanup is tracked separately.
+# flake8: noqa: F403,F405
+
```
Comment (akhilraj9) [inline qa/common/sequence_util.py:47]: Scoped `# flake8: noqa: F403,F405` for the QA-suite `import *` convention. Note it also masks two pre-existing `in0`→`dummy_in0` typos in the string-tensor path — tracked for a proper fix in [TRI-1643](https://linear.app/nvidia/issue/TRI-1643/fix-latent-in0dummy-in0-copy-paste-bugs-in-sequence-utilpy-string).

**Review by pskiran1 — APPROVED** [review]

**Review by Vinya567 — APPROVED** [review]
LGTM

**Review by yinggeh — APPROVED** [review]
